pkg/catalog/catalog.go

```diff
@@ -87,13 +90,40 @@ func VersionExists(client *occlient.Client, componentType string, componentVersi
 }
 
 // getDefaultBuilderImages returns the default builder images available in the
-// openshift namespace
+// openshift and the current namespaces
 func getDefaultBuilderImages(client *occlient.Client) ([]CatalogImage, error) {
-	imageStreams, err := client.GetImageStreams(occlient.OpenShiftNameSpace)
-	if err != nil {
-		return nil, errors.Wrap(err, "unable to get Image Streams")
+	var imageStreams []imagev1.ImageStream
+	currentNamespace := client.GetCurrentProjectName()
+	var err error
+
+	// Fetch imagestreams from default openshift namespace
+	openshiftNSImageStreams, e := client.GetImageStreams(occlient.OpenShiftNameSpace)
```

Comment:
make it err instead of e

Reply:
you can reuse err

Reply:
No please check https://github.com/redhat-developer/odo/pull/618/files/f397bb7134a87fa371a3616810a3c750186595fa#r213039396

Reply:
use something else but not `e` :P

Reply:
possibleError seems good? :)